clc/paths/paths_test.go

```diff
@@ -0,0 +1,78 @@
+package paths_test
+
+import (
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/hazelcast/hazelcast-commandline-client/clc/paths"
+	. "github.com/hazelcast/hazelcast-commandline-client/internal/check"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/it/skip"
+)
+
+func TestHomeDir_Unix(t *testing.T) {
+	skip.If(t, "os = windows")
```

Comment:
Is it possible to add tests for Windows?

Reply:
Added Windows tests at: https://github.com/hazelcast/hazelcast-commandline-client/pull/160/commits/b9fd102d81c6100a8d5beef5b705fbf6f42f464e Note that these should be run manually ATM.